arlo-client/src/components/CreateAudit.tsx

```diff
@@ -60,18 +71,37 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
   return (
     <Wrapper>
       <img height="50px" src="/arlo.png" alt="Arlo, by VotingWorks" />
-      <Button
-        verticalSpaced
```

Comment:
This line seems to have been lost when I rebased. Do we want it, or is it right that it's gone? @MorganLove

Reply:
@jonahkagan That works with it being gone. I just compared with and without and it looks better without. Thanks for checking! 👍